### ISSUE TITLE
Test suite missing files on Hackage

### ISSUE THREAD

**Issue by snoyberg** [mrkkrp/htaglib#2]
```
Test suite tests: RUNNING...
Getters:
  checking file: "audio-samples/sample.flac": [Failed]
ERROR: InvalidFile "audio-samples/sample.flac"
  checking file: "audio-samples/sample.mp3": [Failed]
ERROR: InvalidFile "audio-samples/sample.mp3"
  checking file: ("audio-samples/sample.flac",FLAC): [Failed]
ERROR: InvalidFile "audio-samples/sample.flac"
  checking file: ("audio-samples/sample.mp3",MPEG): [Failed]
ERROR: InvalidFile "audio-samples/sample.mp3"
Setters:
  checking file: "audio-samples/sample.flac": [Failed]
ERROR: audio-samples/sample.flac: copyFile: does not exist (No such file or directory)
  checking file: "audio-samples/sample.mp3": [Failed]
ERROR: audio-samples/sample.mp3: copyFile: does not exist (No such file or directory)
  checking file: ("audio-samples/sample.flac",FLAC): [Failed]
ERROR: audio-samples/sample.flac: copyFile: does not exist (No such file or directory)
  checking file: ("audio-samples/sample.mp3",MPEG): [Failed]
ERROR: audio-samples/sample.mp3: copyFile: does not exist (No such file or directory)

         Test Cases  Total
 Passed  0           0
 Failed  8           8
 Total   8           8
Test suite tests: FAIL
Test suite logged to: /home/stackage/work/logs/nightly/htaglib-0.1.0/test-run.out
```

**Comment by mrkkrp** [comment]
Ah, audio samples not included. I'll hotfix it soon and put it as version 0.1.1. Thanks for the report.

**Comment by mrkkrp** [comment]
@snoyberg, Fixed, [`htaglib-0.1.1`](https://hackage.haskell.org/package/htaglib) on Hackage now includes the samples. I'm closing this for now, feel free to reopen if this fails again.
